x/gamm/pool-models/internal/cfmm_common/lp.go

```diff
@@ -25,7 +25,7 @@ func CalcExitPool(ctx sdk.Context, pool types.CFMMPoolI, exitingShares sdk.Int,
 	var refundedShares sdk.Dec
 	if !exitFee.IsZero() {
 		// exitingShares * (1 - exit fee)
-		oneSubExitFee := sdk.OneDec().SubMut(exitFee)
+		oneSubExitFee := sdk.OneDec().Sub(exitFee)
```

Comment:
not actually a bug (as oneDec is a new copy), but LGTM.

Anyways not on a hot path